### PR TITLE
Adds gem to sort cards

### PR DESCRIPTION
**Pull request by beerlington**
https://github.com/AgilionApps/LaunchPad-UI/pull/37

### CODE REVIEW

File: spec/fabricators/tile_fabricator.rb [hunk 1]
```diff
@@ -13,3 +13,31 @@
   project
 end
 
+Fabricator(:launch, from: :tile, class_name: 'Tile') do
```
Comment (beerlington) [inline spec/fabricators/tile_fabricator.rb:16]: @patrickberkeley I wanted to point out that fabricators need to all be defined in the same file in order for inheritance to work correctly in the specs. Without this, the attributes from the parent are not inherited.

Comment (patrickberkeley) [inline spec/fabricators/tile_fabricator.rb:16]: :+1: Thanks for the info.

**Comment by patrickberkeley** [comment]
LGTM

**Pull request merged**